helmfile.d/20-fitbit.yaml

```diff
@@ -41,6 +41,8 @@ releases:
         values: [{{ .Values.server_name }}]
       - name: serverName
         value: {{ .Values.server_name }}
+      - name: authUrl
+        value: {{ .Values.server_name }}/managementportal/oauth
```

Comment:
I think it's better to fix this in the helm chart instead of here

Reply:
You mean that the helmchart will derive the authUrl from the _server_name_ value? From the perspective of the helm chart this does not make sense to me, since the Authentication Service in theory might live off cluster.

Reply:
Ah but I can make the chart default to the URL with _server_name_, but use `.Values.authUrl` when specified. Does that seem like a good idea? This has as a consequence that `authUrl` will be null in the default values.

Reply:
It sounds elegant to me, because we keep the customizations at the helm chart level.

The bit about the default null value seems like a small thing to me :)

Reply:
Yes defaulting to `server_name` when `authUrl` isn't specified is a good solution to this.